src/MICore/Logger.cs

```diff
@@ -72,73 +71,63 @@ public static Logger EnsureInitialized(HostConfigurationStore configStore)
 
         public static void LoadMIDebugLogger(HostConfigurationStore configStore)
         {
-            if (s_logger == null)
-            { 
-                if (CmdLogInfo.enabled)
-                {   // command configured log file
-                    s_logger = HostLogger.GetLoggerFromCmd(CmdLogInfo.logFile, CmdLogInfo.logToOutput);
-                }
-                else
-                {   // use default logging
-                    s_logger = configStore.GetLogger("EnableMIDebugLogger", "Microsoft.MIDebug.log");
```

Comment:
>EnableMIDebugLogger [](http://example.com/codeflow?start=54&length=19)

If we are going to get rid of this, we should update the wiki and remove the script to set it.

Reply:
Sounds good. My thought is we have much better ways of enabling logging so we could remove this method.